.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
```

Comment:
As of the moment, all of the files which are _actually_ part of this library are indented with 2 spaces.